### DISCUSSION TITLE
New version requires "Read browsing history" permission?

### DISCUSSION THREAD

**Discussion by jffaust** [OneKeyHQ/app-monorepo#5592] [General]
Why does the latest version (5.1) of the OneKey browser extension require browsing history?

![image](https://github.com/user-attachments/assets/2e6ab72f-0891-48b5-a1f8-c3c36f83c6b8)

**Comment by originalix** [comment]
Thanks for feedback. The incorrect permission request was the cause of the issue. This has been fixed in the latest version, and this permission is no longer required.

Fix in #5595
